src/vs/base/browser/ui/tree/abstractTree.ts

```diff
@@ -823,6 +828,8 @@ class FindWidget<T, TFilterData> extends Disposable {
 		this.width = width;
 		this.right = clamp(this.right, 0, Math.max(0, width - 212));
 		this.elements.root.style.right = `${this.right}px`;
+		this.top = clamp(this.top, 0, parseInt(window.getComputedStyle(this.elements.root).height));
```

Comment:
Apart from `window.getComputedStyle` being computationally expensive, the widget's height is fairly well-known, so let's just hardcode this to 24px or so.